src/signal_scope/mainwindow.ui

```diff
@@ -27,10 +27,10 @@
     <property name="title">
      <string>File</string>
     </property>
-    <addaction name="ActionOpen"/>
+    <!-- <addaction name="ActionOpen"/> -->
```

Comment:
is this removing the open command?  Maybe instead of removing it we can rename it to "Open JSON".  Perhaps the JSON stuff can be retired soon, but I think we're still using it a little bit right now.  Perhaps we could do this on a separate pull request.